aspnet/mvc/views/view-components.rst

```diff
@@ -18,57 +27,91 @@ New to ASP.NET MVC 6, view components are similar to partial views, but they are
- 
-One use of a view component could be to create a login panel that would be displayed on every page with the following functionality:
+- A login panel that would be rendered on every page and show either the links to log out or log in, depending on the log in state of the user
 
-- If the user is not logged in, a login panel is rendered.
-- If the user is logged in, links to log out and manage account are rendered.
-- If the user is in the admin role, an admin panel is rendered.
+A `view component <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html>`__ consists of two parts, the class (typically derived from  ``ViewComponent``) and the result it returns (typically a view). Like controllers, a view component can be a POCO, but most developers will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
 
-You can also create a view component that gets and renders data depending on the user's claims. You can add this view component view to the layout page and have it get and render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+Creating a view component
+---------------------------
+
+This section contains the high level requirements to create a view component. Later in the article we'll examine each step in detail and create a view component.
 
-A view component consists of two parts, the class (typically derived from  ``ViewComponent``) and the Razor view which calls methods in the view component class. Like controllers, a view component can be a POCO, but most users will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
+The view component class
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 A view component class can be created by any of the following:
 
-- Deriving from `ViewComponent`.
-- Decorating the class with the ``[ViewComponent]`` attribute, or deriving from a class with the ``[ViewComponent]`` attribute.
-- Creating a class where the name ends with the suffix *ViewComponent*.
+- Deriving from `ViewComponent`
+- Decorating a class with the ``[ViewComponent]`` attribute, or deriving from a class with the ``[ViewComponent]`` attribute
+- Creating a class where the name ends with the suffix *ViewComponent*
 
-Like controllers, view components must be public, non-nested, non-abstract classes.
+Like controllers, view components must be public, non-nested, and non-abstract classes.
 
-Examine the ViewComponent class
---------------------------------
+A view component class:
 
-- Examine the *src\\TodoList\\ViewComponents\\PriorityListViewComponent.cs* file:
+- Fully supports constructor `Dependency Injection <https://docs.asp.net/en/latest/fundamentals/dependency-injection.html>`__
+- Does not take part in the controller lifecycle, which means you can't use action filters in a view component
```

Comment:
> you can't use action filters in a view component

you can't use `` :doc:`filters </mvc/controllers/filters>` `` in a view component